xmake/rules/c++/modules/xmake.lua

```diff
@@ -50,7 +50,6 @@ rule("c++.build.modules")
 
 -- build modules
 rule("c++.build.modules.builder")
-    set_sourcekinds("cxx")
```

Comment:
How do we generate module dependencies from cpp files without this ?

Reply:
please wait, I'll improve it

Reply:
I have fixed it.